concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-402	402	kiss; sniff	375	SNIFF	4
+Marrison-1967-909-403	403	kitchen	1365	KITCHEN	2
+Marrison-1967-909-404	404	knee	1371	KNEE	2
+Marrison-1967-909-405	405	knife	1352	KNIFE	2
+Marrison-1967-909-406	406	knock	2396	KNOCK	2
+Marrison-1967-909-407	407	knot (in string)	760	KNOT	4
+Marrison-1967-909-408	408	know	1410	KNOW (SOMETHING)	2
+Marrison-1967-909-409	409	ladder	1464	LADDER	2
+Marrison-1967-909-410	410	ladle	2657	LADLE	2
+Marrison-1967-909-411	411	lake	624	LAKE	2
+Marrison-1967-909-412	412	lame	320	LAME	2
+Marrison-1967-909-413	413	lamp	625	LAMP	2
+Marrison-1967-909-414	414	land	626	LAND	2
+Marrison-1967-909-415	415	language	1307	LANGUAGE	2
+Marrison-1967-909-416	416	late	477	LATE	2
+Marrison-1967-909-417	417	laugh	1355	LAUGH	2
+Marrison-1967-909-418	418	law	1062	LAW	2
+Marrison-1967-909-419	419	lazy	1564	LAZY	2
+Marrison-1967-909-420	420	lead	43	LEAD (GUIDE)	2
+Marrison-1967-909-421	421	leaf	628	LEAF	2
+Marrison-1967-909-422	422	learn	504	LEARN	2
+Marrison-1967-909-423	423	leather	629	LEATHER	2
+Marrison-1967-909-424	424	leech	2273	LEECH	2
+Marrison-1967-909-425	425	left-hand	2182	LEFT HAND	2
+Marrison-1967-909-426	426	leg	1297	LEG	2
+Marrison-1967-909-427	427	lemon	3091	LEMON	2
+Marrison-1967-909-428	428	leopard	1139	LEOPARD	2
+Marrison-1967-909-429	429	leper			
+Marrison-1967-909-430	430	letter	1612	LETTER	2
+Marrison-1967-909-431	431	lick	319	LICK	2
+Marrison-1967-909-432	432	lie (deceive)	601	LIE (MISLEAD)	4
+Marrison-1967-909-433	433	lie down	215	LIE DOWN	2
+Marrison-1967-909-434	434	life	210	LIFE	2
+Marrison-1967-909-435	435	lift	501	LIFT	2
+Marrison-1967-909-436	436	light (bright)	679	BRIGHT	4
+Marrison-1967-909-437	437	light (ignite)	737	LIGHT (IGNITE)	4
+Marrison-1967-909-438	438	light (of weight)	1052	LIGHT (WEIGHT)	2
+Marrison-1967-909-439	439	lightning	1654	LIGHTNING	2
+Marrison-1967-909-440	440	like (as)	1558	SIMILAR	4
+Marrison-1967-909-441	441	lip	478	LIP	2
+Marrison-1967-909-442	442	lip (lower)			2
+Marrison-1967-909-443	443	listen	1608	LISTEN	2
+Marrison-1967-909-444	444	little	1246	SMALL	2
+Marrison-1967-909-445	445	live (to be alive)	1422	BE ALIVE	4
+Marrison-1967-909-446	446	liver	1224	LIVER	2
+Marrison-1967-909-447	447	lizard	632	LIZARD	2
+Marrison-1967-909-448	448	locust	810	LOCUST	2
+Marrison-1967-909-449	449	loincloth	1965	LOINCLOTH	2
+Marrison-1967-909-450	450	long	1203	LONG	2
+Marrison-1967-909-451	451	look at	1819	LOOK	2
+Marrison-1967-909-452	452	look for	1468	LOOK FOR	2
+Marrison-1967-909-453	453	loom	1787	LOOM	2
+Marrison-1967-909-454	454	lord	3153	LORD	4
+Marrison-1967-909-455	455	lose	1660	LOSE	2
+Marrison-1967-909-456	456	louse	1392	LOUSE	2
+Marrison-1967-909-457	457	love	923	LOVE	2
+Marrison-1967-909-458	458	low	721	LOW	2
+Marrison-1967-909-459	459	lungs	688	LUNG	2
+Marrison-1967-909-460	460	mad (insane)	1899	MAD	4
+Marrison-1967-909-461	461	maize	506	MAIZE	2
+Marrison-1967-909-462	462	make	734	MAKE	2
+Marrison-1967-909-463	463	man (male)	2106	MALE PERSON	2
```

Comment:
maybe better http://concepticon.clld.org/parameters/1554, as MAN is the male person which is not young ("erwachsen"). Maybe check the reflexes in the URL above: if it links many othe rlists with "man/male" to this 1554, use 1554 MAN.